File: python/run demo/Market Data/AvgPrice.py
from python.spot import mexc_spot_v3


market = mexc_spot_v3.mexc_market()

# Enter parameters in JSON format in the "params", for example: {"symbol":"BTCUSDT", "limit":"200"}
# If there are no parameters, no need to send params
params = {
    "symbol": "🤮🐰💗❤️🌞💤📖⛰️USDT",
}
AvgPrice = market.get_avgprice(params)
print(AvgPrice)

File: go/run demo/Market Data/AvgPrice.go
package main

import (
	spotList "demo/spot"
	"fmt"
)

//说明：
//在params中输入json格式的参数,如:`{"symbol":"BTCUSDT",	"limit":"200"}`
//如果没有参数则输入:""

// 现货参数
var params string = `{"symbol":"️BTCUSDT"}`

func main() {

	//接口调用
	AvgPrice := spotList.AvgPrice(params)
	fmt.Println("返回信息:", AvgPrice)

}
